configure.ac

```diff
@@ -2141,6 +2147,11 @@ AC_ARG_ENABLE([ed25519],
     [ ENABLED_ED25519=$enableval ],
     [ ENABLED_ED25519=no ]
     )
+AC_ARG_ENABLE([ed25519-stream],
```

Comment:
Did you consider using `--enable-ed25519=stream` and document in ed25519 help?

Reply:
:arrow_up: